src/views/forms/FormTransferToken/FormTransferToken.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+    <div class="FormTransferToken">
+        {{ mosaicId }}
+        <FormWrapper>
+            <ValidationObserver v-slot="{ handleSubmit }" ref="observer" slim>
+                <form onsubmit="event.preventDefault()">
+                    <div class="create-nft-mosaic-form">
+                        <div class="create-nft-mosaic-left">
+                            <div v-if="showTransactionActions" class="transfer-actions">
+                                <a @click="isImportTransactionUriModalVisible = true">
+                                    <!--<Icon type="md-arrow-round-down" />-->{{ $t('import_transaction_uri') }}
+                                </a>
+                            </div>
+
+                            <!-- Transaction signer selector -->
+                            <SignerSelector
+                                v-if="!hideSigner"
+                                v-model="formItems.signerAddress"
+                                :signers="signers"
+                                @input="onChangeSigner"
+                            />
+
+                            <!-- Transfer recipient input field -->
+                            <RecipientInput v-model="formItems.recipientRaw" style="margin-bottom: 0.5rem;" @input="onChangeRecipient" />
```

Comment:
It seems to me that these styles are unnecessary for display.

Reply:
Why?

Reply:
The component already has a bottom margin. In this case, there are too many margins between the fields.